data/event/event.go

```diff
@@ -8,8 +8,9 @@ const (
 // Event is the data type to represent arbitrary, one-off messages for key
 // activities on a system.
 type Event struct {
-	Summary  string `json:"summary"`
-	Category string `json:"category,omitempty"`
+	Summary    string                 `json:"summary"`
+	Category   string                 `json:"category,omitempty"`
+	Attributes map[string]interface{} `json:"attributes,omitempty"`
```

Comment:
what are attributes used for? could you add a comment pls

Reply:
also a small doc will be great

Reply:
will do 👍

Reply:
I pushed a new commit with some comments and an example in code, is this ok or you had something else in mind.